websites/F/Flyff Universe/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.14",
+  "apiVersion": 1,
+  "author": {
+    "id": "142786385387061249",
+    "name": "KHRM"
+  },
+  "service": "Flyff Universe",
+  "description": {
+    "en": "Flyff Universe is a new Free to Play web MMORPG in 3D. Play the best Flyff version on your PC, Mac, Phone or Tablet. Fly for Fun, anywhere, anytime!"
+  },
+  "url": "universe.flyff.com",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/WTLdzZL.png",
+  "thumbnail": "https://imgur.com/WTLdzZL.png",
```

Comment:
Thumbnail should be a banner, promotional image, or screenshot of the website homepage. (doesn't have to be 512x512)